### DISCUSSION TITLE
Is it possible to override (disable) the up-keep option of a container when clicking a specific link?

### DISCUSSION THREAD

**Discussion by hansipete** [unpoly/unpoly#528] [Q&A]
I have this basic layout of my project
```
<nav>
<div up-overlay>
<div up-main="root" up-keep>
  <a up-target="[up-overlay]">
</div>
<footer>
```

Most of the interaction happens from a page called `/team` where the up-main container is filled with links from team members. When clicking them the detail page will open, overlapping ca 50% of the page.

A few other pages will not use the overlay, but I'd like to fill the main container with other content. At the same time I'd like to keep the `up-keep` parameter on that, because it prevents flickering of content when closing the overlay.

Is there a way to have links like `<a href up-follow up-override-keep>`?

Thank you

**Comment by adam12** [comment]
There is an [`[up-use-keep]`](https://unpoly.com/a-up-follow#up-use-keep) attribute that you could possibly set on the `<a>` element. I am not sure if it will work for your use case (I've personally never used it).

**Reply by hansipete** [reply]
Shame on me, I didn't scroll down to the very bottom :-)

This is working perfectly. Thank you so much!
